WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewController.swift

```diff
@@ -1,22 +1,29 @@
 import UIKit
 import SwiftUI
 import Yosemite
+import Experiments
```

Comment:
Nice point here @iamgabrielma :) For me it is always a good practice to inject the dependency `FeatureFlagService` because it ensures testability, even though in some cases as in this one we won't test it (we are talking about a `UIViewController` subclass here). Furthermore, even if we use the Service Locator directly, my bet is that we would have to import `Experiments`, since the interface of `FeatureFlagService` stays there. On a minor note, having the `featureFlagService` as a property makes the if statement easier to read:

```
guard featureFlagService.isFeatureFlagEnabled(.paymentsHubMenuSection) else {
            return nil
        }
```